src/common/libutil/parse_size.h

```diff
@@ -31,6 +31,17 @@
  */
 int parse_size (const char *s, uint64_t *vp);
 
+/* Format 'size' as a human readable string using suffixes documented
+ * above for parse_size(). Note that due to use of doubble precision
```

Comment:
typo "doubble" . "double"

Reply:
i was curious why the typo checker didn't catch this .... "doubble"  is apparently the name of a dating app!

Reply:
🤦‍♂️